src/Composer/AppAutoloadPlugin.php

```diff
@@ -31,4 +31,14 @@ public function activate(Composer $composer, IOInterface $io): void
             $rootPackage->setAutoload(['psr-4' => ['App\\' => 'src/']]);
         }
     }
+
+    public function deactivate(Composer $composer, IOInterface $io)
+    {
+        // Nothing to do here
```

Comment:
We should remove the autoload entry here, shouldn't we?

Reply:
the `deactivate()` method seems to be relevant if one registers a plugin class, so that class is unloaded before a plugin is installed. That's not the case with the `contao/manager-plugin`.